website/docs/user_guides/overview_of_chaos_dashboard.md

```diff
@@ -0,0 +1,150 @@
+
+Because we only have a single pod at this example, we can click next directly. For more detail about `Scope`, you can view [Define the Scope of Chaos Experiment](experiment_scope).
+
+#### Target
+
+In this overview we will select a `NetworkChaos` with `delay` `10ms latency`, that's very simple. Just select the tab of `Network`, choose action to Delay, then fill `Latency` to `10ms`.
+
+Below is the result:
+
+![Create Experiment Target](/img/chaos-dashboard/create-experiment-target.png)
+
+Click the next button you can set the experiment schedule by unchecking the `immediate` radio button, fill two fields `cron` and `duration` you want, then all steps are finished. You can submit this experiment right now.
+
+As you can see, no need to write spec by YAML, no need to apply by hand, the chaos dashboard can help you quickly try different chaos.
+
+**But what if you have already defined some experiments by YAML files, and what if you want to use them by dashboard?**
+
+**Or, you have already created or archived some experiments and you want to duplicate or recover the experiment?**
+
+The right half is for you. Currently, we have these three options:
+
+- Load From Existing Experiments
+- Load From Existing Archives
+- Load From YAML File
+
+You can try these after the overview.
+
+### Experiment detail
+
+Click the detail button of experiment in the `Experiments` page, we can view our previous created:
+
+![Experiments](/img/chaos-dashboard/experiments.png)
+
+![Experiment Detail](/img/chaos-dashboard/experiment-detail.png)
+
+You can exec `Archive`, `Pause (Start)`, `Update` operation to the experiment, the detail page also shows the timeline and events with this experiment. You can explore more after this overview.
+
+## Events
+
+![Events](/img/chaos-dashboard/events.png)
+
+The events page shows all experiments' events, also with the timeline panel.
+
+### Search Events
+
+If you want to quickly locate some events. You can search at the top right of the events table.
+
+We also define some useful patterns to help you search events:
+
+- `namespace:default` will search events with namespace default.
+- `kind:NetworkChaos` will search events with kind NetworkChaos, you can also type kind:net, it will also work because the search is fuzzy.
+- `pod:xxx` will search events with pod name xxx.
+- `uuid:xxx` will search events with uuid xxx.
+
+All of these patterns can be appended with `experiment name`, for example, you can type `kind:PodChaos xxx` to filter events which name starts with `xxx` with kind PodChaos.
+
+## Archive
+
+To introduce this section, I have already archived the experiment we just defined.
+
+![Archive](/img/chaos-dashboard/archive.png)
+
+We define the content of the archive page as `report`. All of your experiments will not be lost, their final form is archive. If you want to stop an experiment, just archive it.
+
+And you can recover(duplicate) the archive anytime.
+
+## At Last
```

Comment:
This heading is a little bit weird to me. Generally we won't use such a heading.